.travis.yml

```diff
@@ -4,7 +4,16 @@ python:
 sudo: required
 
 install:
-  - make install
+  - make install install-docs
 
 script:
-  - make black check_conftest_imports doctest coverage
+  - make black check_conftest_imports doctest coverage doc-build
+
+deploy:
+  provider: pages
```

Comment:
Can you link to the documentation on this provider, please?  Maybe even as a comment for easy future reference.